.github/workflows/on-pr-debug.yml

```diff
@@ -62,3 +58,29 @@ jobs:
     with:
       sanitize: thread
       server_version: main
+
+  Windows:
+    name: "Windows"
+    runs-on: windows-latest
+    steps:
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v7
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Checkout nats.c
+        uses: actions/checkout@v4
+
+      - name: Build
+        env:
+          VCPKG_BINARY_SOURCES: "clear;x-gha,readwrite"
+        run: |
+          cmake -B build -S . -DCMAKE_C_FLAGS=/W4 -DNATS_BUILD_STREAMING=OFF
+          cmake --build build --config Debug
+
+      - name: Test
+        run: |
+          cd build
+          ./bin/Debug/testsuite
```

Comment:
This does not run the test suite, it just builds the list of tests. Was it intended?

Reply:
yes, that was my intention to have a GH Windows build and improved debugging experience using VS and CLion on Windows. I propose we address testing in a follow up PR.